corehq/apps/es/aggregations.py

```diff
@@ -111,6 +111,32 @@ def counts_by_bucket(self):
         return {b['key']: b['doc_count'] for b in self.normalized_buckets}
 
 
+class StatsResult(AggregationResult):
+
+    @property
+    def count(self):
```

Comment:
i don't ever use these props, so i'm thinking i can just remove or do you think it's useful to keep around so that the interface is defined?